filters/include/pcl/filters/impl/voxel_grid.hpp

```diff
@@ -202,13 +203,18 @@ pcl::getMinMax3D (const typename pcl::PointCloud<PointT>::ConstPtr &cloud,
 
 struct cloud_point_index_idx 
 {
-  unsigned int idx;
+  int idx;
```

Comment:
Could you please explain why this change is needed? Especially considering that line 342 nevertheless instantiates these structures with `unsigned int` cast.

Reply:
I was following the example in the boost::sort library, but I made a test and this change is not needed in fact.

I should probably revert it to make it more clear